notebooks/intro_data_science/Data_Commons_For_Data_Science_Tutorial.ipynb

```diff
@@ -3,10 +3,9 @@
   "nbformat_minor": 0,
   "metadata": {
     "colab": {
-      "name": "Data Commons For Data Science Tutorial.ipynb",
```

Comment:
why the removal of the name?

Reply:
This was not a change I made intentionally, it looks like colab's export metadata no longer includes name compared to 2 years ago.